graphbolt/src/cnumpy.h

```diff
@@ -147,6 +146,68 @@ class OnDiskNpyArray : public torch::CustomClassHolder {
   static inline std::mutex available_queues_mtx_;  // available_queues_ mutex.
   static inline std::vector<int> available_queues_;
 
+  struct QueueAndBufferAcquirer {
```

Comment:
Better to explain how the Acquirer works at the top of the class.
Use class for struct with methods, struct is reserved only for data structure.

Reply:
See #7713

Reply:
Which guideline says that struct is reserved only for C style data structures? If you can point it out, I can read it and learn why this is preferred. I normally like using struct when the first things I am writing are supposed to be public.

Reply:
https://google.github.io/styleguide/cppguide.html#Structs_vs._Classes